DependencyInjection/Configuration.php

```diff
@@ -125,6 +125,14 @@ public function getConfigTreeBuilder()
                             ->defaultTrue()
                             ->info('Enable automatic registration of annotations with JMSDiExtraBundle')
                         ->end()
+                        ->scalarNode('role_admin')
+                            ->defaultValue('ROLE_SONATA_ADMIN')
+                            ->cannotBeEmpty()
```

Comment:
Can't get it: why adding keys with default values (without altering how the bundle works) it's a BC break?